content/Products/RHOBS/configuring-clients-for-observatorium.md

```diff
@@ -0,0 +1,233 @@
+type: Opaque
+stringData:
+  CLIENT_ID: $CLIENT_ID
+  CLIENT_SECRET: $CLIENT_SECRET
+  ISSUER_URL: https://sso.redhat.com/auth/realms/redhat-external
+  URL: "https://observatorium-mst.api$([ -n "$STAGING" ] && echo .stage).openshift.com/api/metrics/v1/$TENANT/api/v1/receive"
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app.kubernetes.io/component: authentication-proxy
+    app.kubernetes.io/name: token-refresher
+    app.kubernetes.io/version: master-2020-12-04-5504078
+  name: token-refresher
+  namespace: openshift-monitoring
+spec:
+  ports:
+    - name: http
+      port: 80
+      targetPort: 8080
+  selector:
+    app.kubernetes.io/component: authentication-proxy
+    app.kubernetes.io/name: token-refresher
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/component: authentication-proxy
+    app.kubernetes.io/name: token-refresher
+    app.kubernetes.io/version: master-2020-12-04-5504078
+  name: token-refresher
+  namespace: openshift-monitoring
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: authentication-proxy
+      app.kubernetes.io/name: token-refresher
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: authentication-proxy
+        app.kubernetes.io/name: token-refresher
+        app.kubernetes.io/version: master-2020-12-04-5504078
+    spec:
+      containers:
+      - args:
+        - --oidc.audience=observatorium-telemeter
+        - --oidc.client-id=\$\(CLIENT_ID\)
+        - --oidc.client-secret=\$\(CLIENT_SECRET\)
+        - --oidc.issuer-url=\$\(ISSUER_URL\)
+        - --url=\$\(URL\)
+        env:
+        - name: CLIENT_ID
+          valueFrom:
+            secretKeyRef:
+              name: token-refresher
+              key: CLIENT_ID
+        - name: CLIENT_SECRET
+          valueFrom:
+            secretKeyRef:
+              name: token-refresher
+              key: CLIENT_SECRET
+        - name: ISSUER_URL
+          valueFrom:
+            secretKeyRef:
+              name: token-refresher
+              key: ISSUER_URL
+        - name: URL
+          valueFrom:
+            secretKeyRef:
+              name: token-refresher
+              key: URL
+        image: quay.io/observatorium/token-refresher:master-2021-02-24-1e01b9c
+        name: token-refresher
+        ports:
+        - containerPort: 8080
+          name: http
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  labels:
+    app.kubernetes.io/component: authentication-proxy
+    app.kubernetes.io/name: token-refresher
+  name: token-refresher
+  namespace: openshift-monitoring
+spec:
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/component: authentication-proxy
+      app.kubernetes.io/name: token-refresher
+  policyTypes:
+  - Ingress
+  ingress:
+  - from:
+    - podSelector:
+        matchLabels:
+          prometheus: k8s
+EOF
+```
+
+## 2. Using Grafana to Read Metrics from Observatorium
```

Comment:
IIRC, this section was a draft and has not been tested, right?

Reply:
I am not sure about this one. I took this as-is from the internal doc.

Reply:
Yes, but this whole section comes after something that says:
```
=== DRAFT BELOW ===
```